packages/eui/src/components/text/text.tsx

```diff
@@ -22,6 +22,10 @@ export type TextSize = (typeof TEXT_SIZES)[number];
 
 export type EuiTextProps = CommonProps &
   Omit<HTMLAttributes<HTMLDivElement>, 'color'> & {
+    /**
+     * Determines the root element
```

Comment:
Minor copy suggestion
```suggestion
     * The HTML element tag to render
```